### ISSUE TITLE
Donations link

### ISSUE THREAD

**Issue by engstk** [rescuezilla/rescuezilla#462]
Hi Shasheen!

Could you consider alternative donation platforms?
Not everyone wants to join Patreon or do recurring donations and you are probably missing out because of this.
Please consider Paypal and similar (or open platforms that work with several donations players, like Open Collective) since it's what most people have available and it's easier.

Best regards.

**Comment by shasheene** [comment]
Several people have asked for this before, but I haven't set up another platform yet.

If you would like to make a one-off or non-Patreon contribution, the only method right now is through PayPal: https://www.paypal.me/shasheene

**Comment by engstk** [comment]
> Several people have asked for this before, but I haven't set up another platform yet.
> 
> If you would like to make a one-off or non-Patreon contribution, the only method right now is through PayPal: https://www.paypal.me/shasheene

Yeah paypal should cover it more widely, you should put it next to the other donation links som more people can see it :)

Have a beer on me and keep up the good work!
